AsyncDisplayKit/ASTableView.mm

```diff
@@ -848,10 +894,15 @@ - (void)rangeController:(ASRangeController *)rangeController didInsertNodes:(NSA
   if (!self.asyncDataSource) {
     return; // if the asyncDataSource has become invalid while we are processing, ignore this request to avoid crashes
   }
-
+  
   BOOL preventAnimation = animationOptions == UITableViewRowAnimationNone;
-  ASPerformBlockWithoutAnimation(preventAnimation, ^{
+  ASPerformBlockWithoutAnimationCompletion(preventAnimation, ^{
     [super insertRowsAtIndexPaths:indexPaths withRowAnimation:(UITableViewRowAnimation)animationOptions];
+  }, ^{
```

Comment:
Added the dispatch calls after the inserts, deletes etc. and the content size reflects the new size after inserting so if a new check if a batch should occur happens shortly after it should have the updated value to check if a batch should happen or not.

This would also remove the need of a new no animation handler.